react.js

```diff
@@ -27,14 +30,15 @@ this.state = {
   dislikes: ["mirrors", "garlic", "wooden stakes"]
 }
 
-
+[name, home, dislikes] = this.state
 
```

Comment:
Destructuring uses a variable declaration and curly braces to unpack the values from the object
const { name, home, dislikes } = this.state